unittests/test_stretching.py

```diff
@@ -4,7 +4,7 @@
 import pytest
 from obspy.signal.invsim import cosine_taper
 
-from noise_module import stretching, stretching_vect
+from noisepy.seis.noise_module import stretching, stretching_vect
```

Comment:
This is great!

Question: Is there a specific reason that there are 2 test folders: test and unittests?

Reply:
Yeah, legacy. The `test` directory used to contain both one-of manual testing scripts and unit tests. I split the unit tests into their own directory to make it easier to run them via pytest during CI. Perhaps I'll rename:
`tests` --> `scripts`
`unittests` --> `tests`
wdyt?

Reply:
I like that better! It's definitely a lot more clear.